strr-web/components/bcros/form-section/property-manager/BusinessDetails.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <div data-test-id="form-business-details-info">
+    <BcrosFormSection :title="t('createAccount.propertyManagerForm.businessDetails')">
+      <div class="flex flex-row justify-between w-full mb-[40px] mobile:flex-col mobile:mb-[16px]">
+        <UFormGroup name="businessName" class="flex-grow">
+          <UInput
+            v-model="businessName"
+            type="text"
+            aria-label="property manager business legal name"
```

Comment:
Might not need area-label, since the button is self explanatory, or if required it should come from translation file.

Reply:
As per discussion, aria-labels have been removed. Eventually work to get aria-label from translation will need to be taken up post MVP, as it is not being done anywhere currently.